packages/schemas/alterations/next-1709086085 -user-password-encrypt-method.ts

```diff
@@ -0,0 +1,36 @@
+import { sql } from 'slonik';
+
+import type { AlterationScript } from '../lib/types/alteration.js';
+
+const alteration: AlterationScript = {
+  up: async (pool) => {
+    await pool.query(sql`
+      alter type users_password_encryption_method add value 'SHA1';
+      alter type users_password_encryption_method add value 'SHA256';
+      alter type users_password_encryption_method add value 'MD5';
+      alter type users_password_encryption_method add value 'BCrypt';
```

Comment:
```suggestion
      alter type users_password_encryption_method add value 'Bcrypt';
```

https://en.wikipedia.org/wiki/Bcrypt